specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -23,6 +23,9 @@ model AcsIncomingCallEventData {
   /** Signed incoming call context. */
   incomingCallContext?: string;
 
+  /** The communication identifier of the user on behalf of whom the call is made. */
+  onBehalfOfCallee?: CommunicationIdentifierModel;
```

Comment:
Should this be required? Or is it expected that it will be omitted for certain events?

Reply:
yeah it will be omitted fpr certain events